sql/share/errmsg-utf8.txt

```diff
@@ -7792,6 +7792,9 @@ ER_COMPRESSION_DICTIONARY_NAME_TOO_LONG
 ER_COMPRESSION_DICTIONARY_DATA_TOO_LONG
   eng "Data for compression dictionary '%-.192s' is too long (max length = %lu)"
 
+ER_COMPRESSION_DICTIONARY_DATA_ZERO_LEN
+  eng "Data for compression dictionary can't be of zero lenght"
```

Comment:
length typo

Reply:
Also add this one as last error message (else it would error number for other codes)

Reply:
Have a dictionary name passed to it, print the name of dictionary